lucien-ios/StartMyCollection/StartMyCollectionViewController.swift

```diff
@@ -10,5 +10,32 @@ import UIKit
 
 final class StartMyCollectionViewController: UIViewController {
     @IBAction func startCollectionButtonPressed(_ sender: UIButton) {
+        let addComicViewController = AddComicViewController()
+        let addComicNavigationController = UINavigationController(rootViewController: addComicViewController)
+
+        // NavigationBar Background
+        addComicNavigationController.navigationBar.backgroundColor = LucienTheme.white
```

Comment:
The majority of this can be handled in `AddComicViewController` itself. Unless it's something that only `StartMyCollectionViewController` knows about, it shouldn't be handled here